docs/client/install-kn.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing the kn CLI"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/install/install-kn
+  - /docs/client/connecting-kn-to-your-cluster
+---
+
+This guide provides details about how you can install the Knative `kn` client (CLI) using various methods.
+
+## Install kn using an executable binary
+
+You can install `kn` by downloading the executable binary for your system and placing it in the system path.
+
+Links to the latest stable executable binary releases are available on the <a href="https://github.com/knative/client/releases" target="_blank">`kn` release page</a>.
+
+
+### Install kn using nightly executable binary
```

Comment:
kn is not only executable at night ;-) It's nightly _built_ so, "by downloading a nightly built executable" would be more appropriate.